error-issue/src/stackdriver_api.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright 2020 The AMP HTML Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Stackdriver} from './types';
+import {GaxiosOptions} from 'gaxios';
+import {GoogleAuth} from 'google-auth-library';
+
+const SERVICE = 'https://clouderrorreporting.googleapis.com';
+const SECONDS_IN_DAY = 60 * 60 * 24;
+
+export class StackdriverApi {
+  private auth = new GoogleAuth({
+    scopes: 'https://www.googleapis.com/auth/cloud-platform',
```

Comment:
nit: move to const

Reply:
Are you referring to the scope string or the `auth` private member?

Reply:
scope string

Reply:
Doen.